### ISSUE TITLE
Excellent tool, why not parsing date < 1900 ? Could it be improved ?

### ISSUE THREAD

**Issue by crouti** [scrapinghub/dateparser#492]
Thanks for you excellent tool!
It is hard to understand that such efforts are limited to date > 1900.
Any simple way to use your tool for dates < 1900 ?
Thanks!

**Comment by perrinjerome** [comment]
I'm just a user here and I don't want to speak in the name of maintainers, but as far as I see dates before 1900 are supported in dateparser:
```python
import dateparser
dateparser.parse("300 years ago")
# outputs: datetime.datetime(1719, 1, 30, 1, 47, 46, 655559)
```

A guess about what might be the problem: on python 2, using strftime is not supported with date < 1900 ( https://bugs.python.org/issue1777412 )

**Comment by crouti** [comment]
> I'm just a user here and I don't want to speak in the name of maintainers, but as far as I see dates before 1900 are supported in dateparser:
> 
> ```python
> import dateparser
> dateparser.parse("300 years ago")
> # outputs: datetime.datetime(1719, 1, 30, 1, 47, 46, 655559)
> ```
> 
> A guess about what might be the problem: on python 2, using strftime is not supported with date < 1900 ( https://bugs.python.org/issue1777412 )

Yes, you are right, so the solution would be to use Python 3 correct ?
